src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,15 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    /*
+     * Returns a string that contains all the information of a person except tags
+     * 
+    */
+    public default String getPrintableString(Printable... printables){
```

Comment:
Correct implementation of the varags method. As discussed in tutorial, there are many possible placement for this method (ReadOnlyPerson is a reasonable place to "park" the method).